lib/cib/cib_file.c

```diff
@@ -348,11 +348,10 @@ cib_file_perform_op_delegate(cib_t *cib, const char *op, const char *host,
         return -EPROTONOSUPPORT;
     }
 
-    cib->call_id++;
     cib__set_call_options(call_options, "file operation", cib_no_mtime);
 
-    request = cib_create_op(cib->call_id, op, host, section, data, call_options,
-                            user_name);
+    request = cib__create_op(cib, op, host, section, data, call_options,
```

Comment:
This still doesn't get NULL-checked as of this commit but it's fixed as of the next commit... not worried about it now